cg/apps/orderform/excel_orderform_parser.py

```diff
@@ -193,21 +193,20 @@ def get_data_delivery(self, project_type: OrderType) -> str:
         if self.is_from_orderform_without_data_delivery(data_delivery):
             return self.default_delivery_type(project_type)
 
-        if data_delivery == "fastq-qc":
-            return DataDelivery.FASTQ_QC
-        if data_delivery == "fastq-+-analysis":
-            return DataDelivery.FASTQ_ANALYSIS
-        if data_delivery == "analysis-+-scout":
-            return DataDelivery.ANALYSIS_SCOUT
-        if data_delivery == "fastq-+-analysis-+-scout":
-            return DataDelivery.FASTQ_ANALYSIS_SCOUT
-        if data_delivery == "fastq-qc-+-analysis":
-            return DataDelivery.FASTQ_QC_ANALYSIS
-
+        delivery_type_map = {
+            "fastq-qc": DataDelivery.FASTQ_QC,
```

Comment:
Could you also include "no_value": self.default_delivery_type(project_type) in the look.up and get rid of the special if above? And perhaps these keys ( "fastq-qc" etc) could be constants defined only once in cg?

Reply:
I looked a bit more at this code and simplified it a lot. The `is_from_orderform_without_data_delivery` and `default_delivery_type` functions do not seem to be used except here and have to do with custom microsalt orders (which I have never seen). I removed them without causing any pytests to fail.

Reply:
NIce 👍